### PR TITLE
Switched calculated properties to b3

### PR DESCRIPTION
**Pull request by kkrampa**
@biyeun

### CODE REVIEW

File: corehq/apps/domain/templates/domain/internal_calculations.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
     </script>
 {% endblock %}
 
-{% block main_column %}
+{% block page_content %}
     <div class="form form-horizontal">
```
Comment (benrudolph) [inline corehq/apps/domain/templates/domain/internal_calculations.html]: the form controls will also have to be modified: http://getbootstrap.com/css/#forms

**Comment by biyeun** [comment]
@kkrampa you need to migrate the forms styles too see: https://staging.commcarehq.org/styleguide/forms/#b3-migration

File: corehq/apps/domain/templates/domain/internal_calculations.html (hunk 1 above)
Comment (biyeun) [inline corehq/apps/domain/templates/domain/internal_calculations.html]: And if possible, please try to convert this form to a crispy form. Follow the guide here in the Django Form Checklist which shows you how to take an existing Django form and turn it into a Crispy Form. https://staging.commcarehq.org/styleguide/forms/#b3-migration

Comment (kkrampa) [inline corehq/apps/domain/templates/domain/internal_calculations.html]: This form is pretty unusual. It's even not using <form> tag and it's only used to show data on UI not to send something to backend so for me it doesn't make sense to convert this to crispy.

**Comment by kkrampa** [comment]
@biyeun Updated PR and attached screen in description.

File: corehq/apps/domain/templates/domain/internal_calculations.html [hunk 2]
```diff
@@ -39,8 +39,8 @@
     </script>
 {% endblock %}
 
-{% block main_column %}
-    <div class="form form-horizontal">
+{% block page_content %}
+    <div class="form-horizontal">
```
Comment (biyeun) [inline corehq/apps/domain/templates/domain/internal_calculations.html]: `form form-horizontal`

File: corehq/apps/domain/templates/domain/internal_calculations.html [hunk 3]
```diff
@@ -39,8 +39,8 @@
...
         <p><button type="button" id="load-all-btn" class="btn btn-primary">{% trans 'Load EVERYTHING' %}</button></p>
         <hr />
         {% for calc_tag in order %}
```
Comment (biyeun) [inline corehq/apps/domain/templates/domain/internal_calculations.html]: @kkrampa can you please take a look at http://getbootstrap.com/migration/
This is the first link in the migration guide and covers all the css classes that need to be renamed. For instance, you forgot to rename `control-group` to `form-group` or `controls-row` to `row`, all input classes to `form-control` and then wrap the corresponding fields in controls inside separate div tags. eg

```
<div class="controls row">
<div class="col-sm-8">
  ...field data...
</div>
<div class="col-sm-4">
 ...load property button...
</div>
</div>
```
I'll clarify the migration guide now to make sure this step isn't missed in the future.

**Comment by kkrampa** [comment]
@biyeun updated

File: corehq/apps/domain/templates/domain/internal_calculations.html [hunk 4]
```diff
@@ -39,24 +39,25 @@
...
-            <div class="control-group calc-group">
-                <label class="control-label">{{ calcs|dict_lookup:calc_tag }}</label>
-                <div class="controls control-row">
-                    <div class="pull-right" style="width: 200px;">
+            <div class="form-group calc-group">
+                <div class="controls row">
+                    <label class="control-label col-sm-2 col-md-1">{{ calcs|dict_lookup:calc_tag }}</label>
```
Comment (biyeun) [inline corehq/apps/domain/templates/domain/internal_calculations.html]: `col-md-1` is actually tiny on a medium sized screen. I would remove this class

**Comment by biyeun** [comment]
just one last tiny thing and then g2g :+1:

**Comment by kkrampa** [comment]
@biyeun Updated

**Comment by biyeun** [comment]
awesome

**Pull request merged**